src/els_utils.erl

```diff
@@ -1,12 +1,11 @@
 -module(els_utils).
 
--export([ find_document/1
+-export([ find_header/1
         , find_module/1
-        , find_module/2
         , fold_files/4
+        , lookup_document/1
```

Comment:
Why the difference in naming for this function? Why not `find_document/1`?

Reply:
The idea was to distinguish between a lookup and a search operation.